### DISCUSSION TITLE
Does GLTFast support point texture sampling?

### DISCUSSION THREAD

**Discussion by wonkee-kim** [atteneder/glTFast#292] [Q&A] [answered]
Hi,
I just wonder if GLTFast supports point texture sampling so we can load very small minecraft-like textures.
If it doesn't, do you have a plan to support it someday?

**Comment by atteneder** [comment, marked as answer]
hi @wonkee-kim ,

Yes, that should work!

glTF supports texture samplers that let you choose the filters for minification and magnification. If you select `9728` (=`Nearest`) for both, you should have the desired effect (point filtering).

Sources:

https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#_filtering
https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-sampler

Let me know if you experience issues.

**Reply by wonkee-kim** [reply]
Oh! Perfect!! That's really great!
